project_scrum/models/project_task.py

```diff
@@ -0,0 +1,30 @@
+from odoo import api, fields, models
+from odoo.exceptions import ValidationError
+
+
+class ProjectTask(models.Model):
+    _inherit = "project.task"
+
+    sprint_id = fields.Many2one(
+        comodel_name="project.sprint",
+        string="Sprint",
+        tracking=True,
+        domain="['|', ('project_id', '=', False), ('project_id', '=', project_id)]",
```

Comment:
I think the sprint selection should only display unfinished records. Otherwise, over time, the list could become too large. What do you think?
```suggestion
        domain="[('state', '!=', 'done'), '|', ('project_id', '=', False), ('project_id', '=', project_id)]",
```

Reply:
Done !